.github/workflows/main.yml

```diff
@@ -25,43 +24,32 @@ jobs:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
 
-
       # Runs a single command using the runners shell
-      - name: Install GNU Guix
-        uses: PromyLOPh/guix-install-action@v1
-
-      # Runs a set of commands using the runners shell
-      - name: Build image
-        run: scripts/build_docker_image.sh
-      - name: Upload pack (Docker)
-        uses: actions/upload-artifact@v2
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
         with:
-          name: mobilizon-reshare-docker
-          path: docker-image.tar.gz
-  publish:
+          python-version: "3.9"
+      - name: Install dependencies
+        run: |
```

Comment:
consider to move this to a script for consistency

Reply:
thanks, fixed